gen/dcos-config.yaml

```diff
@@ -459,6 +459,7 @@ package:
       MESOS_LAUNCHER_DIR=/opt/mesosphere/active/mesos/libexec/mesos
       MESOS_EXECUTOR_ENVIRONMENT_VARIABLES=file:///opt/mesosphere/etc/mesos-executor-environment.json
       MESOS_EXECUTOR_REGISTRATION_TIMEOUT=10mins
+      MESOS_RECONFIGURATION_POLICY=additive
```

Comment:
I can't find this flag in `docs/configuration/agent.md`, perhaps it only got added to `agent/flags.*`?

Reply:
You're correct, the corresponding review isn't merged yet: https://reviews.apache.org/r/64450/

Naturally I'm hoping to get it committed as soon as possible, but with the current 1.11 frenzy going on I'm not sure how soon that will be.